src/main/java/edu/cornell/kfs/concur/batch/service/impl/ConcurBatchUtilityServiceImpl.java

```diff
@@ -47,6 +47,13 @@ public String getConcurParameterValue(String parameterName) {
         return parameterValue;
     }
     
+    @Override
+    public boolean getConcurParameterBooleanValue(String parameterName) {
```

Comment:
Would isConcurParameterEnabled be a more suitable name?

Reply:
ok